src/app/authentication/eula/authentication-eula.component.html

```diff
@@ -3,7 +3,9 @@
     <div class="container">
       <div class="row">
         <div class="col-md-12 ml-auto mr-auto mt-5 pt-5 eula-page">
-          <div [innerHTML]="eulaText" class="lighter-link-color"></div>
+          <div class="lighter-link-color">
+            {{eulaText}}
```

Comment:
EULA is not interpreted anymore.
<img width="1145" alt="2021-10-04_09-56-33" src="https://user-images.githubusercontent.com/25266945/135814319-b1d7dfa1-2abd-4ccd-8fab-a5213d262968.png">

Reply:
here innerHTML could be used as soon as the eula is trusted source :)
I'll do the change

Reply:
Yes, indeed, we are the only ones that can change it, delivered in the repo.
No endpoint can change this content.

Reply:
Fixed ;)